grammars/ruby.cson

```diff
@@ -1441,6 +1441,40 @@
     ]
   }
   {
+    'begin': '(?=(?><<[-~]("?)((?:[_\\w]+_|)XML)\\b\\1))'
+    'comment': 'Heredoc with embedded xml'
+    'end': '(?!\\G)'
+    'name': 'meta.embedded.block.xml'
+    'patterns': [
+      {
+        'begin': '(?><<[-~]("?)((?:[_\\w]+_|)XML)\\b\\1)'
+        'beginCaptures':
+          '0':
+            'name': 'punctuation.definition.string.begin.ruby'
+        'contentName': 'source.xml'
```

Comment:
The scopeName of language-xml is [text.xml](https://github.com/atom/language-xml/blob/master/grammars/xml.cson#L1).